README.md

```diff
@@ -80,20 +84,13 @@ After the 16 week course, students have a number of options:
 
 **We believe all of these can co-exist.**
 
-This proposes that a ***commercial-focussed organisation*** is formed
-- run by [@nelsonic] and [@iteles] in the immediate term -
+This proposes that a ***commercial-focussed organisation*** is formed - run by [@nelsonic] and [@iteles] in the immediate term -
```

Comment:
I think its worth getting people's feedback on the *name* of the ***commercial-focussed organisation*** ... see: https://github.com/nelsonic/ateam/issues/13